test/integration/provider/aws.go

```diff
@@ -73,7 +72,7 @@ func DescribeMachines(machineClass *v1alpha1.MachineClass, secretData map[string
 }
 
 // DescribeInstancesWithTag describes the instance with the specified tag
-func DescribeInstancesWithTag(tagName string, tagValue string, machineClass *v1alpha1.MachineClass, secretData map[string][]byte) ([]string, error) {
+func getOrphanedVMs(tagName string, tagValue string, machineClass *v1alpha1.MachineClass, secretData map[string][]byte) ([]string, error) {
```

Comment:
You can name it as `getOrphanesInstances` as `Instances` is the term AWS uses. Also, match the function name in the comment.